README.md

```diff
@@ -1,5 +1,127 @@
 # Predix UI
------------------------------------------------
 
-## Overview
-Predix UI is the Predix UI teams' website, in Github pages format.
+This repo hosts predix-ui.com, the website for GE's Predix Design System. The site includes component demos, design and development guides, and additional tools for using the Predix Design System.
+
+## How to run on your machine
+
+### Install tools
+
+If you don't have them already, you'll need node, bower, and gulp to be installed globally on your machine:
+
+1. Install [node](https://nodejs.org/en/download/). This includes npm -- the node package manager.
+2. Install [bower](https://bower.io/) globally: `$ npm install bower -g`
+3. Install [gulp](http://gulpjs.com/) globally: `$ npm install gulp-cli -g`
+
+### Clone the project and install dependencies
+
+1. Clone the project to your machine and open the folder:
+	```
+	$ git clone https://github.com/predix-ui/predix-ui.github.io.git
+	$ cd predix-ui.github.io
+	```
+2. Install dependencies:
+
+	```
+	$ npm install
+	$ bower install
+	```
+
+### Developing locally
+
+As you are developing, you can see your changes by running `gulp serve` from your command line. This will build the project files, serve the site locally, and automatically refresh the site as you make changes.
+
+### Running a local build
+
+When you think you are ready to deploy your changes, you can build the site locally to test how the changes will behave in production. The localBuild task runs additional tasks that simulate how the site will be built for production. Many of these tasks take a significant amount of time to run, so you should only run this command once you want test that your changes are ready to push to production.
+
+To run a local build:
+  ```
+  $ gulp localBuild
+  $ cd build/default
+  $ npm serve
```

Comment:
just `serve` (or any other static web server). You'll need to explain how to install it above, though `npm -g i serve` etc.